prombench/manifests/prombench/benchmark/4_node-exporter.yaml

```diff
@@ -110,10 +110,11 @@ spec:
       nodeSelector:
         node-name: prometheus-{{ .PR_NUMBER }}
       containers:
-      - image: quay.io/prometheus/node-exporter:v0.16.0
+      - image: quay.io/prometheus/node-exporter:v0.18.0
         args:
         - "--path.procfs=/host/proc"
         - "--path.sysfs=/host/sys"
+        - "--path.rootfs=/host/root"
```

Comment:
There are quite a few unnecessary things here. we could use `--collector.filesystem.ignored-mount-points` and `--collector.filesystem.ignored-fs-types` flags to get rid of those(whichever suites). (from the Disk Space Used Panel)

![di](https://user-images.githubusercontent.com/12918431/87236073-e38e8100-c401-11ea-89aa-5b22c29c7201.png)

Reply:
Added the ignore flags some disks